tidb-cloud/tidb-cloud-password-authentication.md

```diff
@@ -82,7 +82,7 @@ If you forget your password, you can reset it by email as follows:
 
 > **Note:**
 >
-> This section applies only when you [sign up](https://console.tidb.io/free-trial) for TiDB Cloud with emails and passwords. If you sign up for TiDB Cloud with Google, GitHub, or Microsoft SSO, you can enable MFA on your chosen identity management platform.
+> This section applies only when you [sign up](https://console.tidb.io/free-trial?provider_source=alicloud) for TiDB Cloud with emails and passwords. If you sign up for TiDB Cloud with Google, GitHub, or Microsoft SSO, you can enable MFA on your chosen identity management platform.
```

Comment:
```suggestion
> This section applies only when you [sign up](https://console.tidb.io/signup?provider_source=alicloud) for TiDB Cloud with emails and passwords. If you sign up for TiDB Cloud with Google, GitHub, or Microsoft SSO, you can enable MFA on your chosen identity management platform.
```

Reply:
Just keep it as it is. Users can sign up via both links.